api/config/cdr_versions_local.json

```diff
@@ -10,7 +10,6 @@
     "releaseNumber": 1,
     "numParticipants": 946237,
     "cdrDbName": "cdr",
-    "elasticIndexBaseName": "cdr",
-    "publicDbName": "public"
```

Comment:
Does removing this property break the Cdr updater? https://github.com/all-of-us/workbench/blob/master/api/tools/src/main/java/org/pmiops/workbench/tools/UpdateCdrVersions.java#L34

Reply:
Tested - still works. publicDbName is nullable so it's not a problem.